cookbook/file-upload.md

```diff
@@ -11,7 +11,7 @@ First you need an amber project generated with [Amber CLI](../guides/create-new-
 ```ruby
 class SomeController < ApplicationController
   def file_upload
-    image = env.params.files["image1"]
```

Comment:
Interestingly it looks like this was borrowed from Kemal docs. There are definitely a few differences between how kemal and amber handle files. Let me know if anything else seems to work differently. I don't think it's necessary to check of filename is a string with amber. Someone might have changed that though.

Reply:
Working fine without the check. I will report back if that changes. I will try and contribute as much as I can.

Reply:
If you wouldn't mind removing that check and reference to kemal from this doc before I merge the PR that would be great.